### PR TITLE
Replace Travis with GHA

### PR DESCRIPTION
**Pull request by MKodde**
1. The daily sec-check and the test-integration was added
2. Travis tests have been taken down

Note that I used a test-integration workflow from another project that is not yet able to run as a PHP 8 upgrade is needed first. That will be done in the next PR.

### CODE REVIEW

File: .github/workflows/daily-security-check.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
```
Comment (parijke) [inline .github/workflows/daily-security-check.yml]: checkout v4?

File: .github/workflows/daily-security-check.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      - name: Check for node-yarn project
+        id: check_node_yarn
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "yarn.lock"
+      - name: Setup node
+        if: steps.check_node_yarn.outputs.files_exists == 'true'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
```
Comment (parijke) [inline .github/workflows/daily-security-check.yml]: Is this node version still correct?

**Review by parijke — CHANGES_REQUESTED** [review]
versions to be reveiwed in GHA

File: .github/workflows/daily-security-check.yml (hunk 2 above)
Comment (MKodde) [reply .github/workflows/daily-security-check.yml]: Replaced this with the optimized reusable D S Check

File: .github/workflows/daily-security-check.yml (hunk 1 above)
Comment (MKodde) [reply .github/workflows/daily-security-check.yml]: See comment below.

**Review by parijke — APPROVED** [review]

**Pull request merged**